app/services/message_employee_matcher.rb

```diff
@@ -14,4 +14,15 @@ def run
   def day_count
     message.days_after_start
   end
+
+  def match_time(time_zone)
+    employee_current_hour = Time.current.in_time_zone(time_zone).hour
+    employee_current_minute = Time.current.in_time_zone(time_zone).min
+
+    employee_current_hour == message.time_of_day.hour && employee_current_minute == message.time_of_day.min
+  end
+
+  def message_already_sent?(employee)
```

Comment:
also, what about changing this to `message_not_already_sent?` -- the `!` works, too, but since we're only using this method for one purpose, it might be nice to just have it return what we want rather than the opposite (am I making sense? it's early here in PST...)